markdown/how.md

```diff
@@ -3,17 +3,87 @@
 
 ## OpenStack
 
+<!-- Note -->
+
+Our cloud infrastructure runs on OpenStack, and we use it for both the infrastructure of the learning platform and our lab environments.
+
+With the native installation, we used Openstack Heat to deploy our learning platforms to OpenStack VMs. After transitioning to Docker-based deployment, i.e., containerizing the services and running them on Kubernetes, we use Opnetsack magnum to deploy our Kubernetes cluster.
+
+Additionally, we run a self-paced, interactive learning platform, meaning we provide the learners with access to arbitrarily complex, realistic distributed environments on demand where they can learn things by actually doing it and OpenStack Heat comes in handy. 
+Using a Heat template, we can define an exactly reproducible, self-contained environment consisting of, say, ten servers in three networks connected with two routers and arbitrarily involved configurations for each server. 
```

Comment:
Here, in order to not confuse your audience, you must emphasize that although the approach to deploy labs also uses Heat, it is entirely different from the Heat-driven approach to deploy the Open edX infrastructure. Also, reiterate that although we have replaced the Heat-driven approach *to deploy the platform* with one that interacts with Kubernetes, we have *not* replaced the Heat-driven approach to deploy labs.